embuilder.py

```diff
@@ -54,6 +54,8 @@
     'libc++-wasmexcept',
     'libc++-noexcept',
     'libc++-ww-noexcept',
+    'libc++-debug-noexcept',
+    'libc++-debug-ww-noexcept',
```

Comment:
Does this increase the emsdk build size significantly? Though this seems important enough to justify an increase...

Reply:
Each of the extra libraries adds about 8Mb of uncompressed size, out of a total size of about 1.2gb.  Seems worth it to me.

Reply:
Agreed.